resources/grafana.yaml

```diff
@@ -142,6 +142,23 @@ metadata:
 data:
   rollups: |
     [
+      {
+        "retention": "medium",
+        "measurement": "system"
```

Comment:
Is this supposed to not have a trailing comma?

Reply:
As far as I remember in json there should be no comma before next closing bracket "}"

Reply:
https://jsonlint.com/ says that
`{ "a": 1, "b": 2 }`
is valid :man_shrugging:

Reply:
[RFC](https://tools.ietf.org/html/rfc7159#section-4) says:
```
object = begin-object [ member *( value-separator member ) ]
               end-object
```
as you can see the last member is explicitly added without value-separator.

Reply:
If you extract the JSON snippet and run through a JSON validator it will fail.

Reply:
![image](https://user-images.githubusercontent.com/965809/70744678-75c28f80-1d22-11ea-886a-e8f006a7996d.png)

Reply:
ah there's two missing commas... fixing it